### PR TITLE
Fixed Responsive issue in Project/GetNighthawk

### PR DESCRIPTION
**Pull request by NikhilSharma03**
Signed-off-by: Nikhil Sharma <nikhilsharmamusic2000@gmail.com>

**Description**
- There is a responsive issue in the project/getnighthawk page for smaller screens.

** Screenshot **
Screenshot from 2021-08-06 12-57-17

## After Changes
- aligned the text in the middle

** Screenshot **
Screenshot from 2021-08-06 13-03-14

**Notes for Reviewers**
Let me know if there are any other changes we can make.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]
LGTM!

**Comment by NikhilSharma03** [comment]
@warunicorn19 I also tried making changes in project/ meshery
## Current State
Screenshot from 2021-08-06 14-13-27

## After Changes

**Comment by l5io** [comment]
🚀 Preview for commit d8e241be390fc018fbe96a02a30f57023922b962 at: https://610cf9c414cf627bc0b6b547--layer5ng.netlify.app

**Comment by warunicorn19** [comment]
@NikhilSharma03, The change definitely looks better. Why don't you put in a PR for that, and then we'll see if it needs more changes. Thanks Nikhil.

**Comment by NikhilSharma03** [comment]
> @NikhilSharma03, The change definitely looks better. Why don't you put in a PR for that, and then we'll see if it needs more changes. Thanks Nikhil.

Yup i created. Thank you :smile:

File: src/sections/Projects/GetNighthawk/gnh.style.js [hunk 1]
```diff
@@ -170,8 +170,13 @@ const Gnhwrapper = styled.section`
         .hero-section{
             padding: 3.125rem 0;
             .hero-text{
-                width: 50%;
-            }
+                width: 90%;
+		text-align:center;
```
Comment (Nikhil-Ladha) [inline src/sections/Projects/GetNighthawk/gnh.style.js:174]: Do make it center-align from `767px` itself.

Comment (NikhilSharma03) [reply src/sections/Projects/GetNighthawk/gnh.style.js:174]: Yeah Sure.

**Comment by NikhilSharma03** [comment]
@Nikhil-Ladha Can you please review?

**Comment by l5io** [comment]
🚀 Preview for commit 223f9b9432d2fec39b66e9745fc59f2845a21790 at: https://610d1c024e6ed79b8c2d7b26--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**